in_toto/model.go

```diff
@@ -491,13 +506,26 @@ by the step are constrained by the artifact rules in the step's
 ExpectedMaterials and ExpectedProducts fields.
 */
 type Step struct {
-	Type            string   `json:"_type"`
-	PubKeys         []string `json:"pubkeys"`
-	ExpectedCommand []string `json:"expected_command"`
-	Threshold       int      `json:"threshold"`
+	Type                   string                  `json:"_type"`
+	PubKeys                []string                `json:"pubkeys"`
+	CertificateConstraints []CertificateConstraint `json:"certConstraints"`
```

Comment:
Maybe `certConstraints` should be `cert_constraints` to match `expected_command` beneath this.